src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,69 @@
+package pages;
+
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+
+public class LoginPage {
+
+    public LoginPage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(how = How.ID, using = "identifierId")
+    public WebElement userNameField;
+
+    @FindBy(how = How.CLASS_NAME, using = "CwaK9")
+    public WebElement nextButton;
+
+    @FindBy(how = How.NAME, using = "password")
+    public WebElement passwordField;
+
+    @FindBy(how = How.CLASS_NAME, using = "goog-imageless-button-content")
+    public WebElement calendarButton;
+
+    @FindBy(how = How.XPATH, using = "//SPAN[@class='gb_bb gbii']")
```

Comment:
If possible try to avoid using XPATH. It is significantly slower than using id or css locators.